src/main/java/org/broadinstitute/dsm/db/ViewFilter.java

```diff
@@ -261,10 +261,20 @@ public static void addQueryCondition(@NonNull Map<String, String> queryCondition
         }
         else {
             String queryCondition = "";
-            if (queryConditions.containsKey("ES")) {
-                queryCondition = queryConditions.get("ES");
+            if (filter != null && filter.getParentName() != null &&
+                    filter.getParticipantColumn().getTableAlias() != null && "proxy".equals(filter.getParticipantColumn().getTableAlias())) {
+                if (queryConditions.containsKey("proxy")) {
```

Comment:
having `proxy` string as a constant in `Filter` would be convenient, for later usage in somewhere else as well.

Reply:
Good point! I will add it to ElasticSearchUtil because it is a ES object